Add Tanchiky FA news post (#5018)
ppy/osu-wiki: news/2021-03-03-new-featured-artist-tanchiky.md

Check out this [osu!mania mapset](https://osu.ppy.sh/beatmapsets/685580) by [Shima Rin](https://osu.ppy.sh/users/6089608), this [osu!taiko mapset](https://osu.ppy.sh/beatmapsets/1247020) hosted by [Genjuro](https://osu.ppy.sh/users/3196091), or this [loved osu! mapset](https://osu.ppy.sh/beatmapsets/774760) by [Cassu2](https://osu.ppy.sh/users/2587961)!

<audio controls>
    <source src="https://assets.ppy.sh/artists/144/POWER/Tanchiky%20-%20ENERGY%20SYNERGY%20MATRIX.mp3" type="audio/mpeg">
</audio>

### Tanchiky & Street - Empty flow

Check out this [osu! mapset](https://osu.ppy.sh/beatmapsets/1364215) by [NyarkoO](https://osu.ppy.sh/users/6622567) and [DTM9 Nowa](https://osu.ppy.sh/users/5428909) from the Mappers' Guild!

<audio controls>
    <source src="https://assets.ppy.sh/artists/144/REDOUT/Tanchiky%20&%20Street%20-%20Empty%20Flow.mp3" type="audio/mpeg">
</audio>

### Tanchiky - Sanshi Suimei

Check out this [osu! mapset](https://osu.ppy.sh/beatmapsets/811685) by [nika mika](https://osu.ppy.sh/users/3878832)!

<audio controls>
    <source src="https://assets.ppy.sh/artists/144/Songs/Tanchiky%20-%20Sanshi%20Suimei.mp3" type="audio/mpeg">
</audio>

---

> Once you're done playing through those awesome maps, head over to [**Tanchiky**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/144) for a truckload more songs, all pre-timed and ready to be taken under your reigns.

Reviewer: ```suggestion
Once you're done playing through those awesome maps, head over to [**Tanchiky**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/144) for a truckload more songs, all pre-timed and ready for you to take the reins.
```
